kotlintest-assertions-arrow/build.gradle

```diff
@@ -2,4 +2,8 @@ dependencies {
     compile project(":kotlintest-assertions")
     compile 'io.arrow-kt:arrow-typeclasses:0.8.1'
     compile 'io.arrow-kt:arrow-data:0.8.1'
-}
\ No newline at end of file
+    testCompile 'io.arrow-kt:arrow-test:0.8.1'
+    kapt    'io.arrow-kt:arrow-annotations-processor:0.8.1'
```

Comment:
Will this leak into downstream projects, or is it only used when building KotlinTest itself?

Reply:
Only when building itself, once the artifact is published it's not needed anymore.